include/dlpack/dlpack.h

```diff
@@ -126,7 +126,8 @@ typedef enum {
 } DLDataTypeCode;
 
 /*!
- * \brief The data type the tensor can hold.
+ * \brief The data type the tensor can hold. Currently, non-native endianness isn't
```

Comment:
How about change to data type is assumed to follow the native endian-ness. An explicit error message should be raised when attempting to export an array with non-native endianness

Reply:
Changed this way, thanks @tqchen!